src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -1276,7 +1290,15 @@ void ImportStoreIndirect(int token)
 
         void ImportStoreIndirect(TypeDesc type)
         {
-            throw new NotImplementedException($"{nameof(ImportStoreIndirect)} not implemented");
+            ClearPendingPrefix(Prefix.Unaligned);
+            ClearPendingPrefix(Prefix.Volatile);
+
+            var value = Pop();
+            var address = Pop();
+
+            Check(!address.IsReadOnly, VerifierError.ReadOnlyIllegalWrite);
+            CheckLoadIndirect(address, type);
```

Comment:
I do not think it is correct to use `CheckLoadIndirect` for verification of stores. It does the assignability check in a wrong direction for store.

E.g.

```
ref object x
string y

x = y // verifiable
y = x; // not verifiable
```